front/admin_ui/app.py

```diff
@@ -78,7 +79,8 @@ def draw_graph(width, height):
                 query_pending_jobs_button = gr.Button("Run")
                 pending_jobs_query_result_df = gr.DataFrame()
             with gr.Tab("Refresh dataset"):
-                refresh_type = gr.Textbox(label="Processing step type", placeholder="split-first-rows-from-streaming")
+                job_types = [processing_step.job_type for processing_step in PROCESSING_GRAPH.get_topologically_ordered_processing_steps()]
```

Comment:
Note that the admin app ProcessingGraph is not synced with the production one in general, but I think it's fine

Reply:
I think that the solution is to automatize the deployment of the space from the CI.